libafl/src/stages/calibrate.rs

```diff
@@ -161,21 +194,38 @@ where
                 history_map.resize(map_len, O::Entry::default());
             }
 
-            for (first, (cur, history)) in
-                map_first.iter().zip(map.iter().zip(history_map.iter_mut()))
+            for (idx, (first, (cur, history))) in map_first
+                .iter()
+                .zip(map.iter().zip(history_map.iter_mut()))
+                .enumerate()
             {
                 if *first != *cur && *history != O::Entry::max_value() {
                     *history = O::Entry::max_value();
-                    unstable_entries += 1;
+                    unstable_entries.push(idx);
```

Comment:
how is it ensured that an unstable edge is not put twice or more in the vector list?

Reply:
unstable_entries (Vec<usize>) would have duplicates, but when it is inserted into existing.unstable_entries (HashSet<usize>), the duplicate are resolved, because it's a set